settings.gradle

```diff
@@ -0,0 +1,4 @@
+include 'impersonation-filter',
+        'impersonation-filter-bundle-impersonation-filter-bundle'
```

Comment:
I might've just called this project `impersonation-filter-bundle` Save yourself a lot of typing